### PR TITLE
TL translations - 1093 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
For this contribution, I have completely modified two different files. Please help me review and correct them. Thank you!

### CODE REVIEW

File: book/09-git-and-other-scms/1-git-and-other-scms.asc [hunk 1]
```diff
@@ -27,12 +26,12 @@ include::sections/client-p4.asc[]
...
-=== Migrating to Git
+=== Paglilipat sa Git
 
-(((Migrating to Git)))
-If you have an existing codebase in another VCS but you've decided to start using Git, you must migrate your project one way or another.
-This section goes over some importers for common systems, and then demonstrates how to develop your own custom importer.
-You'll learn how to import data from several of the bigger professionally used SCM systems, because they make up the majority of users who are switching, and because high-quality tools for them are easy to come by.
+(((Paglilipat sa Git)))
+Kung mayroon kang isang umiiral na codebase sa ibang VCS ngunit ikaw ay nakapagpasyang magsimulang gumamit ng Git, kailangan mong ilipat ang iyong proyekto sa isang paraan o iba.
+Ang seksyong ito ay dadako sa iilang mga importer para sa karaniwang mga sistema, at pagkatapos ay ipapakita kung paano i-develop ang iyong sariling custom na importer.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/1-git-and-other-scms.asc]: custom  to pasadya

File: book/09-git-and-other-scms/sections/import-bzr.asc [hunk 2]
```diff
@@ -4,45 +4,45 @@
...
 $ sudo dnf install bzr-fastimport
 ----
 
-If the package is not available, you may install it as a plugin:
+Kung ang pakete ay hindi maaaring gamitin, maaari mong i-install ito bilang isang plugin:
 
 [source,console]
 ----
-$ mkdir --parents ~/.bazaar/plugins/bzr     # creates the necessary folders for the plugins
+$ mkdir --parents ~/.bazaar/plugins/bzr     # gagawa ng nararapat na mga folder para sa mga plugin
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-bzr.asc]: gagawa  to gumagawa

File: book/09-git-and-other-scms/sections/import-bzr.asc [hunk 3]
```diff
@@ -4,45 +4,45 @@
...
+$ bzr branch lp:bzr-fastimport fastimport   # mag-i-import ng fastimport na plugin
 $ cd fastimport
-$ sudo python setup.py install --record=files.txt   # installs the plugin
+$ sudo python setup.py install --record=files.txt   # mag-i-install ng plugin
 ----
 
-For this plugin to work, you'll also need the `fastimport` Python module.
-You can check whether it is present or not and install it with the following commands:
+Upang paganahin ang plugin na ito, kakailanganin mo ang `fastimport` na modyul ng Python.
+Maaari mong suriin kung ito ay nandoon o hindi at i-install ito gamit ang sumusunod na mga utos:
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-bzr.asc]: nandoon o hindi to nandoon o wala

**Comment by josteinfrancois** [comment]
Thank you for your review @robin-ho . Already updated the files.

**Pull request merged**